### PR TITLE
term: remove "Dvcfile" from install/plugins 

### PR DESCRIPTION
**Pull request by sarthakforwet**
PR references - 
` install/plugins — see also #1420 (review) (on Vim plugin)` from #1445.

### CODE REVIEW

File: content/docs/install/plugins.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ In order to recognize `.dvc` files as YAML in Vim, you should add:
 
 ```vim
 " DVC
-autocmd! BufNewFile,BufRead Dvcfile,*.dvc setfiletype yaml
+autocmd! BufNewFile,BufRead *.dvc setfiletype yaml
```
Comment (jorgeorpinel) [inline content/docs/install/plugins.md:17]: Thanks @sarthakforwet 

@efiop , do you think we should still keep this for legacy projects? (see https://github.com/iterative/dvc.org/pull/1420#pullrequestreview-432694500)

Comment (efiop) [reply content/docs/install/plugins.md:17]: @jorgeorpinel Yes, I would keep it for now.

**Comment by jorgeorpinel** [comment]
OK @sarthakforwet thanks for checking the **Install** section but it seems there's nothing to update here! Closing 🙂

**Comment by jorgeorpinel** [comment]
P/s but feel free to address the last checkboxes in the original issue, e.g. "push cmd ref."

**Pull request closed without merging**